File: internal/content/testdata/bench/java/longest-ones-run-binary.java
class Solution {
    public long longest_ones_run(long n) {
        String binary = Long.toBinaryString(n);
        String[] parts = binary.split("0");
        long maxLen = 0;
        for (String p : parts) {
            if (p.length() > maxLen) maxLen = p.length();
        }
        return maxLen;
    }
}

File: internal/content/testdata/bench/csharp/longest-ones-run-binary.cs
public class Solution {
    public long longest_ones_run(long n) {
        string binary = Convert.ToString(n, 2);
        string[] runs = binary.Split('0');
        long max = 0;
        foreach (var r in runs) {
            if (r.Length > max) max = r.Length;
        }
        return max;
    }
}
